src/vos/vos_dtx.c

```diff
@@ -1568,6 +1568,16 @@ vos_dtx_act_reindex(struct vos_container *cont)
 			int				 count;
 
 			dae_df = &dbd->dbd_active_data[i];
+			if (dae_df->dae_flags & DTX_EF_INVALID)
```

Comment:
It deserves an error or debug message, that would be helpful for debugging if we hit it during test.

Reply:
The DTX_EF_INVALID flag will be set when abort the DTX, not error.